docs/misc/add_new_gene.md

```diff
@@ -184,3 +186,60 @@ Note: The numbers after new_gene must be integers. A different simulation will r
 `python models/ecoli/analysis/variant/doubling_time_histogram.py`
 
 `python models/ecoli/analysis/variant/new_gene_translation_efficiency_heatmaps.py`
+
+---
+<b>Addition of new metabolic genes<br>(in progress)
+
+* In the genome insertion directory ('reconstruction/ecoli/flat/new_gene_data/gene_names') populate the files: 
+
+  * metabolic_reaction_external.tsv - list the reactions from the additional pathway. Specify for each reaction their:
+  
+    * id (e.g. ‘rxnpath1’),  
+
+    * Stoichiometry represented as a dictionary where the keys are the reactants and the products, and the values represent the coefficients for each reactant (positive coefficient) or product (negative coefficient) 
+
+    * Direction for reading the reaction (e.g. L2R) 
+
+    * A list containing the name of the enzyme catalyzing the reactions. At the moment each reaction can only be catalyzed by 1 enzyme. 
+
+    * A list containing the name of the substrate of the reaction. At the moment, each reaction can only have one substrate. 
+
+    * The turnover number of an enzyme (kcat) 
+
+    * The Michaelis Menten constant (KM) 
+
+  * metabolites.tsv - list of the metabolites produced by the reaction. Specify for each of them: 
+
+    * Their ID as shown on Ecocyc (e.g. CPD-11890) 
+
+    * Their common name according to Ecocyc. 
+
+    * A list of synonyms. This can be empty. 
+
+    * Chemical formula (e.g. C11H9) 
+
+    * Molecular charge. 
+
+    * Smiles. 
+
+  * If these files are not empty, the WCM will run the metabolic_reactions_external process. 
+
+A new process was created to represent the metabolism corresponding to these reactions. This new process is based on Michaelis Menten kinetics and it is run at each time step before the Metabolism process. To achieve this, two Python files were added: 
+
+  * reconstruction/ecoli/dataclasses/process/metabolism_external_pathway.py an – This script loads the raw data from the files populated above and stores it in the sim_data object. It also makes it accessible as an instance of the new process class. In this file we also define the methods that represent the Michaelis Menten model. This script is called when runParca is run. 
```

Comment:
Filenames should be wrapped inside backticks (`) to have them formatted in a fixed-width font.